docs/UserGuide.md

```diff
@@ -35,100 +39,194 @@
 > * Items with `...` after them can have multiple instances.
 > * The order of parameters is fixed.
 
-#### Viewing help : `help`
-Format: `help`
 
-> Help is also shown if you enter an incorrect command e.g. `abcd`
- 
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
+#### Changing storage location : `change`
+Changes the data file storage location. <br>
+Format: `change location to PATH`
+
+Examples:
+* `change location to C:\Users\Verbena\Dropbox\WhatNow`<br>
+Changes the data file storage location to C:\Users\Verbena\Dropbox\WhatNow<br>
+<img src="images/UI_Prototype/WhatNowUI-ChangeLocation-Colour.png" width="600">
+
+
+#### Adding a task: `add`
+Adds a task to WhatNow<br>
+Format: `add "DESCRIPTION" [on/by] [today/tomorrow/DAY/DATE] [from/at] [START_TIME] [to/till] [END_TIME] [every] [DAY/day/week/month/year] [low/medium/high]` 
+
+> * All task description should be written within double quotation marks.<br>
+Eg Add �buy eggs�. <br>
+> * The format for date must be *day month year* where year is optional.<br>
+Eg: 10th Oct, 4 November, 11th August 2017. Not 12/12/12. <br>
+> * Time should be in the *12 hour format*.<br>
+Eg: 10am, 6pm, 7.30pm. Not 7:30pm, 2359.
+
+Examples:
+* `add "Do CS2103T tutorial" on 4 Oct 2016 from 10am to 11am every week`<br>
```

Comment:
`add "Do CS2103T tutorial" on 4 Oct 2016 from 10am to 11am every week` and `add "Do CS2103T tutorial" **by** 4 Oct 2016 **at** 10am **till** 11am every week` produces the same results.

However, it is possible to `add "Taylor Swift Concert" on 10 Oct 2016 from 8pm to 10pm` but it does not make sense to `add "Taylor Swift Concert" by 10 Oct 2016 at 8pm till 10pm`. The results will still be the same but it would not make sense to the user to type the second one.

On the other hand, it is possible to `add "Complete add function for CS2103T" by 10 Oct 2016` but it would not make sense for the user to `add "Complete add function for CS2103T" on 10 Oct 2016` if what the user meant is 10 Oct 2016 is the deadline and not the day to complete the add function.

So "on" and "by" can have different meanings although it produces the same results.

As for "from" and "to" and "at" and till", both are interchangeable in meanings and produces the same results. However, we wanted it to be more user-friendly so we did not want to restrict the user since both are possible ways to say the same thing.